### ISSUE TITLE
import locks up

### ISSUE THREAD

**Issue by Audisport9** [GoldenCheetah/GoldenCheetah#2225]
Hi,

I've been trying to import a FIT file and upon selecting save, import locks up. Import never finishes. It just stays at 50%, and "saving..........".
![fit not saving](https://cloud.githubusercontent.com/assets/24303861/20813308/19f99f1a-b7e3-11e6-91fa-69a1c0a8c6c4.jpg)

**Comment by liversedge** [comment]
Is it with this FIT file or /any/ fit file?

Are you running 3.4?

Try that first as FIT file formats have changed quite a lot recently with FIT 2.0.

If there are still problems then send me the file and I'll see if its a bug ?

liversedge gmail com

**Comment by Audisport9** [comment]
It is with any FIT file. This all started happening after i deleted a few activities. 

Yes I'm using V3.4 (W7,64bit)

I noticed that when an activity is deleted via left side bar that the Json file still exists in the activities folder. So I tried deleting this and the FIT file from the Imports folder. This still didn't fix the issue.

I then created a new test athlete to check if the import works and FIT files import correctly. 

I have since deleted my original athlete and started over. Luckily i just started using GC
